root/components/common-macros.tt

```diff
@@ -834,9 +834,9 @@ END -%]
     };
     IF medium.name != '';
         args.medium_name = isolate_text(medium.name);
-        l('{medium_format} {position}: {medium_name}', args);
+        l('<span>{medium_format} {position}:</span> {medium_name}', args);
```

Comment:
I think it'd be good to add something like `class="medium-format-position"` here and refer to that in the CSS, so we don't have to worry about ambiguity later. Also easier to grep where a style is coming from that way.